app/models/balance.rb

```diff
@@ -1,10 +1,24 @@
 class Balance < ActiveRecord::Base
   belongs_to :user
   has_many :charges
+  has_one :money_charged
```

Comment:
you should create just field in the database! not the association called `money_charged`